server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -463,4 +482,8 @@ private ButtonTag createSubmitButton(ProgramEditStatus programEditStatus) {
         .withId("program-update-button")
         .withClasses(ButtonStyles.SOLID_BLUE, "mt-6");
   }
+
+  private DivTag insertSpacingDiv() {
```

Comment:
I think I tried adding the margins directly to the elements and it wasn't behaving the way I expected it to. But I might have been doing something else, so I'll try again to confirm. A follow-up issue will be to edit this admin page to better match the mocks which will use a bunch of USWDS stuff, so I'd like to keep it as simple as possible for now.

Reply:
Ah yeah, the issue is I can't just increase [this default](https://github.com/civiform/civiform/blob/f2fd4b23bc5dda4232b6cee0bcc6f59351edea51/server/app/views/style/BaseStyles.java#L32) because that updates a whole lot of things and makes the fieldsets that contain multiple fields look weird. I could change this to add `.withClass("my-4")` to the end of each of the fields in this class, but I kinda prefer a helper method because it makes it easy to update them all at once. What do you think?

Reply:
Another option could be to add flex to this whole thing and control the space between elements.

Reply:
Oh wait, my brain is not firing on all cylinders, I could add the margin to a const and just reference that in all the places. Lmk what option seems the best to you!

Reply:
that sounds good!

Reply:
I ran into trouble using flexbox/grid because it added space around the headers too which we don't want. I refactored so I'm not using a helper method (because that was weird) and just using the class as a const. I also added the todo!